pygame/load.py

```diff
@@ -0,0 +1,45 @@
+# dvd logo simulator in python
+# jobega was here 
+
+import pygame
+import random
+
+
+pygame.init()
+
+width, height = 800, 600
+screen = pygame.display.set_mode((width, height))
+pygame.display.set_caption("DVD Logo Simulator")
```

Comment:
we may want consistency with the project name on github which is "DVD Player Simulator"